accname/name/comp_host_language_label.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Host Language Label</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<label for="t">label</label>
+<input id="t" data-expectedlabel="label" data-testname="host language: label[for] input[type=text]" class="ex">
+<!-- Todo: test all remaining input types with label[for] -->
+
+<label>
+  <input type="checkbox" data-expectedlabel="label" data-testname="host language: label input[type=checkbox] encapsulation" class="ex">
+  label
+</label>
```

Comment:
I see this fails in chrome because of a white space at the beginning of label, and there is one other failure do to white space at the beginning of the label.... I'm not an expert in accname, but from a quick grep, accname doesn't specify to strip white space -- so it seems like this is an browser implementation detail and we shouldn't surface it as a failure. Should these accname test strip whitespace when comparing labels?

Reply:
Hmm. I recalled that it may have trimmed each step, and trimmed once at the end again, but I acknowledge it's not that way currently. @jnurthen @melsumner @accdc do you remember more? 

Looks like there is a continuing [WhiteSpace project](https://github.com/w3c/accname/projects/1) in AccName that makes this more ambiguous that I recall. 

I agree with trimming until the AccName spec is less ambiguous regarding whitespace.

Reply:
@cookiecrook I looked back through the last 9 years of accname and don't see that ever specified.
I agree that as it stands the resultant accessible label spacing is an implementation detail and should be trimmed of whitespace for the test.

Reply:
https://github.com/web-platform-tests/wpt/pull/39604/commits/2594c299c36b39f966a605a5ff1b4534d49377fd